### PR TITLE
feat: 🎸 add onClick on test status for filter

### PR DESCRIPTION
**Pull request by rustho**
add posibility to change filter by click on teststatus button

### CODE REVIEW

File: src/client/CreeveyView/SideBar/TestsStatus.tsx [hunk 1]
```diff
@@ -31,39 +37,57 @@ export function TestsStatus({
...
+        </span>
+      </Button>
       {' / '}
-      <span>
-        <PauseIcon /> {skippedCount}
-      </span>
+      <Button use="link" narrow onClick={() => handleClick('skipped')}>
+        <span
+          css={css`
+            color: #000000;
```
Comment (wKich) [inline src/client/CreeveyView/SideBar/TestsStatus.tsx]:
```suggestion
            color: #040404;
```

File: src/client/CreeveyView/SideBar/TestsStatus.tsx [hunk 2]
```diff
@@ -31,39 +37,57 @@ export function TestsStatus({
...
+        </span>
+      </Button>
       {' / '}
-      <span>
-        <TrashIcon /> {removedCount}
-      </span>
+      <Button use="link" narrow onClick={() => handleClick('null')}>
+        <span
+          css={css`
+            color: #000000;
```
Comment (wKich) [inline src/client/CreeveyView/SideBar/TestsStatus.tsx]:
```suggestion
            color: #040404;

```

File: src/client/CreeveyView/SideBar/TestsStatus.tsx [hunk 3]
```diff
@@ -20,7 +22,11 @@ export function TestsStatus({
   pendingCount,
   skippedCount,
   removedCount,
+  onClick,
 }: TestsStatusProps): JSX.Element {
+  const handleClick = (status: string): void => {
```
Comment (wKich) [inline src/client/CreeveyView/SideBar/TestsStatus.tsx]: `useCallback`

Comment (rustho) [reply src/client/CreeveyView/SideBar/TestsStatus.tsx]: ууууу, новомодные синтаксисы 🐈🐈🐈

**Pull request merged**